### PR TITLE
Chore: Logo path, edit popup, admin buttons

### PR DESCRIPTION
**Pull request by Andycutter**
## PT link
- https://www.pivotaltracker.com/story/show/160441580

## Tasks
- Add root_path to Venue logo
- Make the edit campaign as new campaign popup
- Make admin buttons appear inline

## View
inlinebuttonsrootpathlogo

### CODE REVIEW

File: app/views/partials/_header.html.haml [hunk 1]
```diff
@@ -2,7 +2,7 @@
     .mui-container
         .mui-row
             .mui-col-md-7.mui-col-sm-7.mui-col-xs-7
-                %a#logo.logo-venue{href: "#"}
+                %a#logo.logo-venue{href: "/."}
```
Comment (tochman) [inline app/views/partials/_header.html.haml]: I would try:
```
= link_to '', id: 'logo', class: 'logo-venue'
```

File: app/views/campaigns/edit.js.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+var html = "<%= escape_javascript(render 'campaigns/editform')%>"
```
Comment (tochman) [inline app/views/campaigns/edit.js.erb]: I'd like you to rename this form to `edit_form`

File: app/views/partials/_header.html.haml [hunk 3]
```diff
@@ -2,7 +2,8 @@
     .mui-container
         .mui-row
             .mui-col-md-7.mui-col-sm-7.mui-col-xs-7
-                %a#logo.logo-venue{href: "/."}
+                %a
+                    = link_to '', root_path, id: 'logo', class: 'logo-venue'
```
Comment (tochman) [inline app/views/partials/_header.html.haml]: Skip the `%a`

**Comment by ESteinman** [comment]
I want a view though...

**Pull request merged**